### PR TITLE
[DOCS] Updates Session View topic with links to Kubernetes dashboard topic

### LINKED ISSUE #2214: [DOCS] [8.4] [AWP] Update Session View doc for Kubernetes data
## Description 

In 8.4, session view will have an updated details panel that displays data captured from Kubernetes. This will make it useful for understanding k8s sessions rather than just typical Linux sessions. The Session View doc should describe the new fields and link to the main K8s visualization doc that describes how to capture this data.

## Acceptance Test Criteria

- Session View doc describes Kubernetes data
- Updated Session View doc has been approved by AWP stakeholders

## Notes

related issues: #2187 #2213

### PR DESCRIPTION
**Pull request by benironside**
Fixes #2214 by adding links to the Session view page that refer users to the Kubernetes dashboard page so they will know that Session view can help them view Kubernetes data.

### CODE REVIEW

File: docs/detections/session-view.asciidoc [hunk 1]
```diff
@@ -31,7 +33,7 @@ fields collected by {endpoint-cloud-sec} when this setting is enabled, refer to
 [float]
 [[open-session-view]]
 === Open Session View
-Session View is accessible from the **Hosts**, **Alerts**, and **Timelines** pages.
+Session View is accessible from the **Hosts**, **Alerts**, and **Timelines** pages, as well as the **Kubernetes dashboard**.
```
Comment (joepeeples) [inline docs/detections/session-view.asciidoc:36]:
```suggestion
Session View is accessible from the **Hosts**, **Alerts**, and **Timelines** pages, as well as the **Kubernetes** dashboard.
```

**Review by joepeeples — COMMENTED** [review]
Added a couple of small edits. Also, you prob already tracked this down, but looks like the PR is currently failing because the referenced anchor `kubernetes-dashboard` hasn't been merged into main yet (it's part of https://github.com/elastic/security-docs/pull/2243, which is still open). 

You could probably just add the changes from this PR to #2243, then close this PR, so that everything related to the Kubernetes dashboard will get merged together. Otherwise this PR will keep failing CI until #2243 gets merged.

**Comment by benironside** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by benironside** [comment]
Following Joe's advice and closing the PR, going to include these changes in #2243

**Pull request closed without merging**